reports/Oregon.md

```diff
@@ -677,6 +691,158 @@ id: or-portland-45
 * https://twitter.com/PredatorFiles/status/1278635513746870276
 
 
+### Police fire on gathered protestors from within Justice Center | July 3rd
+
+Prior to the incident below, where police charged protestors, police inside the Justic Center fired pepper balls at gathered protestors.
+
+tags: shoot, pepper-ball
+
+id: or-portland-47
+
+**Links**
+
+* https://mobile.twitter.com/Johnnthelefty/status/1278966824801931264
+
+
+### Local and Federal Police fire on protestors, shove journalist | July 3rd
+
+Footage shows local and federal police emerge quickly from a building, firing riot rounds and spraying pepper spray. Footage shows a man clearly taking pictures to the side being shoved to the ground.
```

Comment:
maybe ..a man to the side taking pictures being shoved to the ground.  Saying "to the side" in the middle reads strangely